mongonaut/forms/form_mixins.py

```diff
@@ -268,8 +254,8 @@ def get_value(document, field_key):
                 return_data = get_value(document._data.get(current_key), new_key)
             else:
                 # Handeling all other fields and id
-                return_data = (document._data.get(None, None) if current_key == "id" else
-                              document._data.get(current_key, None))
+                return_data = document._data.get(current_key, None)
+                # return_data = (document._data.get(None, None) if current_key == "id" else document._data.get(current_key, None))
```

Comment:
Why this change?  The logic is no longer the same because the `return_data` is now different.